test/test_um2netcdf.py

```diff
@@ -206,124 +212,130 @@ def fake_out_path():
     return "/tmp-does-not-exist/fake_input_fields_file.nc"
```

Comment:
I think these aren't actually used at all any more, are they?

Reply:
Yeah you are right, these shouldn't be needed anymore. I've removed them in d778083